zerver/lib/actions.py

```diff
@@ -2568,6 +2579,19 @@ def gather_subscriptions_helper(user_profile):
             subscribed.append(stream_dict)
         else:
             unsubscribed.append(stream_dict)
+    never_subscribed_stream_ids = [item for item in all_streams_id if item not in sub_unsub_stream_ids]
+    never_subscribed_streams = [x for x in all_streams if x['id'] in never_subscribed_stream_ids]
```

Comment:
Yes, because it's not actually a `Stream` object, it's a dictionary representing a stream.  You could call it `stream_dict` or maybe `ns_stream_dict`.